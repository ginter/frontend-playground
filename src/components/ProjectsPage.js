import React from 'react'
import NewProjectForm from '../containers/NewProjectForm'
import VisibleProjectList from '../containers/VisibleProjectList'

const ProjectsPage = () => (
  <div>
    <NewProjectForm />
    <VisibleProjectList />
  </div>
)

export default ProjectsPage
