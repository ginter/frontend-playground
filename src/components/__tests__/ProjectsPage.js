import React from 'react'
import renderer from 'react-test-renderer'
import { createStore } from 'redux'
import { Provider } from 'react-redux'
import reducer from '../../reducers'

import ProjectsPage from '../ProjectsPage'

describe('ProjectsPage', () => {
  it('renders correctly', () => {
    const store = createStore(reducer);
    const component = renderer.create(
      <Provider store={store}>
        <ProjectsPage />
      </Provider>
    );
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
