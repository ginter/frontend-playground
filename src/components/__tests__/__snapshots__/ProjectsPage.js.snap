exports[`ProjectsPage renders correctly 1`] = `
<div>
  <form
    name="new-project"
    onSubmit={[Function onSubmit]}>
    <input
      defaultValue=""
      name="project-name"
      onChange={[Function onChange]}
      type="text" />
    <input
      type="submit" />
  </form>
  <p>
    No projects found
  </p>
</div>
`;
